charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -133,6 +133,10 @@ data:
                 {{ if $account.readPermissions }}--read-permissions {{ join " --read-permissions " $account.readPermissions }}{{ end }} \
                 {{ if $account.writePermissions }}--write-permissions {{ join " --write-permissions " $account.writePermissions }}{{ end }} \
                 --provider-version v2
+    {{- range $index, $customResource := .Values.kubeConfig.customResources }}
+    $HAL_COMMAND config provider kubernetes account \$PROVIDER_COMMAND {{ $name }} --docker-registries dockerhub \
```

Comment:
`PROVIDER_COMMAND` should always be `edit` given the command above should've `add`ed it in case it was not present before.